packages/eslint-plugin-lit-a11y/lib/rules/iframe-title.js

```diff
@@ -18,27 +18,36 @@ const IframeTitleRule = {
       description: '<iframe> elements must have a unique title property.',
       category: 'Accessibility',
       recommended: false,
+      url:
+        'https://github.com/open-wc/open-wc/blob/master/packages/eslint-plugin-lit-a11y/docs/rules/iframe-title.md',
+    },
+    messages: {
+      iframeTitle: '<iframe> elements must have a unique title property.',
     },
     fixable: null,
     schema: [],
   },
   create(context) {
+    /**
+     * @param {import("parse5-htmlparser2-tree-adapter").Element} element
+     */
+    function isUntitledIframe(element) {
+      return (
+        element.name === 'iframe' &&
+        (!Object.keys(element.attribs).includes('title') || element.attribs.title === '')
```

Comment:
Isnt this check redundant (or 'double')? Checking for either one of these should be enough right?

Reply:
it might be `=== undefined`

i agree this predicate is unwieldy though, will rework